src/xma/include/xma.h

```diff
@@ -57,6 +57,7 @@ int32_t xma_get_default_ddr_index(int32_t dev_index, int32_t cu_index, char* cu_
 void xma_get_session_cmd_load(void);
 
 void xma_enable_mode1(void);//Hidden mode. To allow bulk submission of cu commands
```

Comment:
Consider refactoring` xma_enable_mode1()` and `xma_enable_mode2()` to be 1 method, in case there every is a future "mode3".  For example: 
`xma_enable_mode(unsigned int mode) {}`

Reply:
good suggestion. Keeping them separate because mode1 should be rarely used. And separate debug modes seem better for clarity